include/OpenColorIO/OpenColorTypes.h

```diff
@@ -445,6 +445,8 @@ enum GpuLanguage
     GPU_LANGUAGE_GLSL_1_2,          ///< OpenGL Shading Language
     GPU_LANGUAGE_GLSL_1_3,          ///< OpenGL Shading Language
     GPU_LANGUAGE_GLSL_4_0,          ///< OpenGL Shading Language
+    GPU_LANGUAGE_GLSL_ES_1_0,       ///< OpenGL ES Shading Language
+    GPU_LANGUAGE_GLSL_ES_3_0,       ///< OpenGL ES Shading Language
```

Comment:
No. Please add the new enums at the end to avoid a Public API break (which increases the chance to have the ES support in 2.1; otherwise, it will be 2.2. i.e. 2023!)